plugin-dev/Source/Sentry/Sentry.Build.cs

```diff
@@ -1,16 +1,422 @@
 // Copyright (c) 2022 Sentry. All Rights Reserved.
 
-using System.IO;
 using UnrealBuildTool;
+using System;
+using System.IO;
+using System.Reflection;
+using System.IO;
+using System.Diagnostics;
+using System.Text;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
 
 #if UE_5_0_OR_LATER
 using EpicGames.Core;
 #else
 using Tools.DotNETCommon;
 #endif
 
+public static class DateTimeExtensions
+{
+	public static bool EqualsUpToSeconds(this DateTime dt1, DateTime dt2)
+	{
+		return dt1.Year == dt2.Year && dt1.Month == dt2.Month && dt1.Day == dt2.Day &&
+		       dt1.Hour == dt2.Hour && dt1.Minute == dt2.Minute && dt1.Second == dt2.Second;
+	}   
+}
+public class CMakeTargetInst
+{
+	// Based on UE4CMake with modifications 
```

Comment:
Is there a link to where this came from that you can copy and paste here?

Instead of linking to the file through a branch, use the commit sha (on GitHub, if you press `y` it changes the path from branch to sha automatically.